micro_dl/cli/curator_script.py

```diff
@@ -0,0 +1,160 @@
+# %% script to generate your ground truth directory for microDL prediction evaluation
+# After inference, the predictions generated are stored as zarr store.
+# Evaluation metrics can be computed by comparison of prediction to human proof read ground truth.
+#
+import numpy as np
+import os
+from PIL import Image
+import imagio as iio
+import iohub.ngff as ngff
+import argparse
+
+import micro_dl.inference.evaluation_metrics as metrics
+import micro_dl.utils.aux_utils as aux_utils
+from micro_dl.utils.mp_utils import add_channel
+from waveorder.focus import focus_from_transverse_band
```

Comment:
waveorder needs to be in `lit-monai` environment

Reply:
Done:
```text
$ pip show waveorder
Name: waveorder
Version: 1.0.0rc1
```